### DISCUSSION TITLE
Fantastic job!

### DISCUSSION THREAD

**Discussion by olegsatov** [Acly/krita-ai-diffusion#464] [General]
Hi,

Just wanted to say a huge thank you for this plugin! Please keep up the good work and keep the direction you've chosen, when all the technicalities stay under the hood and only what you need to draw is left on the surface.

Tried other similar plugins (for Photoshop) - tired of how illogical and inconvenient the interface is. Automatic1111 - same story. Things are noticeably better with InvokeAI, but it lacks a lot of functionality for complex drawing, when the image is assembled from individual elements. But your plugin for Krita is exactly what I need!

The logic of working with AI is very well organized, everything is very convenient, you don't stumble over anything while working, everything is always at hand. Bravo!

**Comment by tester4488** [comment]
yes the plugin it's fantastic
